src/index.js

```diff
@@ -412,26 +412,26 @@ class Libp2p extends EventEmitter {
     await this._setupPeerDiscovery()
 
     // Once we start, emit and dial any peers we may have already discovered
-    for (const peerData of this.peerStore.peers.values()) {
-      this.emit('peer:discovery', peerData.id)
-      this._maybeConnect(peerData.id)
+    for (const peer of this.peerStore.peers.values()) {
+      this.emit('peer:discovery', peer.id)
+      this._maybeConnect(peer.id)
     }
   }
 
   /**
    * Called whenever peer discovery services emit `peer` events.
    * Known peers may be emitted.
    * @private
-   * @param {PeerDara} peerData
+   * @param {PeerDara} peer
```

Comment:
typo and I dont think `PeerData` is defined anywhere
```suggestion
   * @param {{ id: PeerId, multiaddrs: Array<Multiaddr>, protocols: Array<string> }} peer
```